whois-api/src/main/java/net/ripe/db/whois/api/rest/SourceResolver.java

```diff
@@ -7,47 +7,44 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
-import static net.ripe.db.whois.common.rpsl.ObjectType.AUT_NUM;
-
 @Component
 public class SourceResolver {
 
     private final AuthoritativeResourceData authoritativeResourceData;
-    private final String mainSourceNameString;
-    private final String nonAuthSourceName;
+    private final CIString mainSource;
+    private final CIString nonAuthSource;
 
     @Autowired
     public SourceResolver(final AuthoritativeResourceData authoritativeResourceData,
-                          @Value("${whois.source}") final String mainSourceNameString,
-                          // TODO [SB]: we need to either use the property or the SourceContext for the nonauth source
-                          @Value("${whois.nonauth.source}") final String nonAuthSourceName) {
+                          @Value("${whois.source}") final String mainSource,
+                          @Value("${whois.nonauth.source}") final String nonAuthSource) {
         this.authoritativeResourceData = authoritativeResourceData;
-        this.mainSourceNameString = mainSourceNameString.toLowerCase();
-        this.nonAuthSourceName = nonAuthSourceName.toLowerCase();
+        this.mainSource = CIString.ciString(mainSource);
```

Comment:
Maybe use static import for ciString

Reply:
I prefer including the class name, so it's clear where it's calling

Reply:
I don't see a clear precedent in the code - it's approximately equal between CIString.ciString() and static import ciString().